phpstan.neon

```diff
@@ -1,25 +1,18 @@
 parameters:
     ignoreErrors:
         # false positive, has annotation type above (@todo recheck for possible ignored positives)
-        - '#Access to an undefined property PhpParser\\Node::\$args#' # 20
-        - '#Access to an undefined property PhpParser\\Node::\$var#' # 2
-        - '#Access to an undefined property PhpParser\\Node::\$name#' # 12
+        - '#Access to an undefined property PhpParser\\Node::\$name#' # 4
```

Comment:
Btw, what the number means? Could you add explanation to first item?

Reply:
Are the number of occurrences, so we can give priority to those ones that have more. I'll add it